client/packages/system/src/InventoryAdjustmentReason/Components/InventoryAdjustmentReasonSearchInput.tsx

```diff
@@ -39,7 +39,7 @@ export const InventoryAdjustmentReasonSearchInput: FC<
   const disabled = data?.totalCount === 0;
   const isRequired = !disabled && stockReduction !== 0;
   const reasonFilter = (reason: InventoryAdjustmentReasonRowFragment) => {
-    if (stockReduction === 0 || !stockReduction) return true;
+    if (stockReduction === 0) return false;
```

Comment:
previously would show all options if there is no adjustment

Reply:
yes, that was the intention. bit weird having the select enabled but empty - could you disable it?

Reply:
that would work nicely, especially now that the reason would be adjusted to `null` when adjustment changes from negative/positive to nothing: 163176cec3c2200547534f26e3321f2a96e355a8